data.js

```diff
@@ -208,6 +208,41 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- Articles wrote \n- Press mentions",
```

Comment:
Articles wrote is still no English, see previous comments

Reply:
deleted completely. That was a bit of a useless kpi